modules/ingress-operator.adoc

```diff
@@ -20,9 +20,12 @@ ifdef::cluster-caps[= Ingress Capability]
 == Purpose
 
 ifdef::cluster-caps[]
+The Ingress Operator provides the features for the `Ingress` capability. Ingress capability is enabled by default.
```

Comment:
I can't seem to comment directly on line 17, but should "Capability" be capitalized?

Reply:
Updated